core/src/main/res/values-az/strings.xml

```diff
@@ -151,6 +151,8 @@
   <string name="pref_update_interval_hours_plural">saat</string>
   <string name="pref_update_interval_hours_singular">saat</string>
   <string name="pref_update_interval_hours_manual">Əl ilə</string>
+  <string name="pref_fast_forward_sum">Sürətli irəli düyməsini tıklandığında irəli tullanmaq üçün saniyə sayı özelleştirin</string>
```

Comment:
FYI: We use Transifex to translate our strings. We only push the english strings, i.e. these strings will be overwritten when we pull the translated strings from transifex.